src/UI/Art/SpriteArtist/PlayerSetupAiArtist.java

```diff
@@ -100,10 +100,12 @@ public static void draw(Graphics g, IController controller, Color playerColor)
     myG.fillRect(infoX, infoY, infoW, infoH);
     myG.setColor(new Color(100, 115, 130));
     myG.drawRect(infoX, infoY, infoW, infoH);
-    BufferedImage infoText = SpriteUIUtils.drawTextToWidth(text, aiInfoZoneWidth - nameplateHeight*2);
-    myG.drawImage(infoText, infoX+infoBuffer, infoY+infoBuffer, null);
 
     // Draw the composed image to the window at scale.
     g.drawImage(image, 0, 0, dimensions.width, dimensions.height, null);
+
+    // TODO: Fix drawTextToWidth to not apply scaling automatically.
```

Comment:
I think this is one of those "hard to document design intent" things, hah.
I suppose it does make sense to only apply scaling when you're actually drawing on the graphics buffer to be shown on screen, though.

Reply:
Yeah, it's much simpler if I can draw in actual pixel-space, and not have to worry about whether a utility function is already scaling for me. Based on how much easier it was to draw these new screens with that philosophy, I want to extend it to all the other screens as well. Not all as part of this PR though, probably.